docs/source/Regions2D_functionality.ipynb

```diff
@@ -0,0 +1,2342 @@
+{
```

Comment:
<p>The title is a bit confusing: </p><p><strong>Platting Sonar and Region </strong>works, but masking should be masking a sonar with a region? Maybe you can split the sections: Plotting Sonar and Region, and next section: Masking Sonar  by Region?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OSOceanAcoustics/echoregions/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>For some reason this caught my eye while I tried to delete the auto-generated emails from GitHubs. Please change "sonar" to echogram. Sonar is an instrument, echogram is the image formed by received sonar echoes. </p>